src/pages/Lend/Filter/FilterSectionSort.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<filter-menu-section :title="title" :open="open" :result-count="resultCount">
+		<ais-sort-by :items="defaultSortIndices">
+			<ul slot-scope="{ items, currentRefinement, refine }">
+				<li v-for="item in items" :key="item.value" :value="item.value" class="sort">
+					<a href="#"
+						:class="{ 'selected': item.value === currentRefinement }"
+						@click.prevent="refine(item.value)">
+						{{ item.label | capitalize }}
+					</a>
+				</li>
+			</ul>
+		</ais-sort-by>
+	</filter-menu-section>
+</template>
+
+<script>
+import { AisConfigure, AisSortBy } from 'vue-instantsearch';
+import FilterMenuSection from '@/pages/Lend/Filter/FilterMenuSection';
+
+export default {
+	components: {
+		AisConfigure,
+		AisSortBy,
+		FilterMenuSection,
+	},
+	filters: {
+		capitalize(value) {
```

Comment:
@mcstover Should we move this globally? There may be another use for it in `SelectedRefinements.vue` as the label 'female / male' should be upper case.